cookie_fix.py

```
@@ -1,53 +1,55 @@
-        playwright_cookies = []
-        for cookie in browser_cookies:
-            # 处理 sameSite: null/空字符串 -> 'Lax'，其他值标准化首字母大写
-            same_site = cookie.get('sameSite')
-            if same_site is None or same_site == '':
+    # 从CLI读取浏览器导出的Cookie - 保留 print 用于用户交互
+    print("请粘贴浏览器导出的Cookie JSON (连续敲击两次回车(Enter)结束输入):")
+    lines = list(iter(input, ''))
+    content = '\n'.join(lines)
+
+    if content == '':
+        logger.error("✗ 输入为空，请检查输入内容")
```

Ruff (0.14.13)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)